zpls_echograms/README.md

```diff
@@ -1,104 +1,193 @@
+
+**Example 1:**
+```
+# set default data directories and .xml file 
+GA_DATA="./GA/ZPLSG_sn55067/DATA"
+GA_PROC="./GA/ZPLSG_sn55067/processed"
+GA_XML="./GA/ZPLSG_sn55067/DATA/201610/16101417.XML"
+
+# set the site, the script will look at the mooring configuration and apply the 
+# tilt angle correction (15 degrees for GA02HYPM_Upper) and the theoretical 
+# depth (150 m for GA02HYPM_Upper). 
+SITE="GA02HYPM_Upper"
+
+# convert and process the data
+python zpls_echogram.py -s $SITE -d $GA_DATA -o $GA_PROC -xf $GA_XML -dr 20161115 20161116 -zm AZFP
+```
+The code will process the data from 2016-11-15 through 2016-11-16. The `.nc` files and `.png` plot will be stored under
+`./GA/ZPLSG_sn55067/processed/20161115-20161116/`.
 
 **Example 2:**  
-```python3    zpls_echogram.py    ./GI/ZPLSG_sn55067/DATA    ./GI/ZPLSG_sn55067/DATA/201610/16101417.XML    201611    ./processed_folder```
+```
+python zpls_echogram.py -s $SITE -d $GA_DATA -o $GA_PROC -xf $GA_XML -dr 201611 -zm AZFP
+```
 
-The code will process the data from 11/01/2016 to 11/30/2016. The .nc file and .png plot will be stored at ./processed_folder.
+The code will process the data for the entire month of November, from 2016-11-01 through 2016-11-30. The `.nc` files 
+and `.png` plot will be stored under `./GA/ZPLSG_sn55067/processed/20161101-20161130/`.
 
 **Example 3:**  
-```python3    zpls_echogram.py    ./GI/ZPLSG_sn55067/DATA    ./GI/ZPLSG_sn55067/DATA/201610/16101417.XML    201611    ./processed_folder    -ss    GI02HYPM_Upper```
+```python zpls_echogram.py -s $SITE -d $GA_DATA -o $GA_PROC -xf $GA_XML -dr 201611 -zm AZFP -tc 10```
 
-Once a subsite is specified, the code will look at the mooring configuration and apply the corrected tilt angle (15 degrees for GI02HYPM_Upper) and the theoretical depth (150 m for GI02HYPM_Upper).
+Change the default tilt correction angle to 10 degrees via the optional `-tc` flag.
 
 **Example 4:**  
-```python3    zpls_echogram.py    ./GI/ZPLSG_sn55067/DATA    ./GI/ZPLSG_sn55067/DATA/201610/16101417.XML    201611    ./processed_folder    -ss    GI02HYPM_Upper    -dd    160```
-
-If the instrument is deployed at 160m instead of the the theoretical depth of 150m. The optional flag -dd 160 will overwrite the mooring configuration.
-
-**Example 5:**  
-```python3    zpls_echogram.py    ./GI/ZPLSG_sn55067/DATA    ./GI/ZPLSG_sn55067/DATA/201610/16101417.XML    201611    ./processed_folder    -ss    GI02HYPM_Upper    -cr    -120    -20    -dd    160```
-
-The optional flag -cr will set the colorbar range from -120 to -20.
-
-**Output Naming Convention:**  
-The naming convention is:
+```python zpls_echogram.py -s $SITE -d $GA_DATA -o $GA_PROC -xf $GA_XML -dr 201611 -zm AZFP -dd 160```
 
-subsite + deployment_number + serial_number + date
+If the instrument is deployed at 160 m, instead of the theoretical depth of 150 m. The optional flag `-dd` 160 will 
+overwrite the mooring configuration.
```

Comment:
Suggest:
The optional flag **`-dd 160`** will 
**override** the mooring configuration.

Reply:
Changes made